apps/web/app/api/subscribers/route.ts

```diff
@@ -43,9 +44,12 @@ export async function POST(request: Request) {
       resendKey: process.env['RESEND_KEY'],
       subscriberId: subscriber.id,
     })
-    new Tracker(process.env['MIXPANEL_KEY']).track(Events.NewSubscriber, {
-      distinct_id: subscriber.id,
-    })
+    new Tracker(process.env['NEXT_PUBLIC_MIXPANEL_KEY']).track(
+      Events.ConfirmedSubscriber,
+      {
+        distinct_id: email,
+      }
+    )
```

Comment:
This is wrong.

This should track the NewSubscriber, not the confirmed.

The confirmed should be tracked when the user effectively confirms (clicks on the email link)